src/sugar3/activity/activity.py

```diff
@@ -1212,7 +1317,7 @@ def __get_filters_cb(self):
         }
         filter_dict = dbus.Dictionary(filters, signature='sv')
         logging.debug('__get_filters_cb %r' % dbus.Array([filter_dict],
-                      signature='a{sv}'))
+                                                         signature='a{sv}'))
```

Comment:
I have an 'AutoPep8Fix' package in my sublime which fixes the Pep8 errors on 'Ctrl+S'. I'll make sure to disable it.